src/chart/main/style.h

```diff
@@ -245,6 +245,7 @@ struct Axis
 	Tick ticks;
 	Guide guides;
 	Interlacing interlacing;
+	Param<Gfx::Length> split;
```

Comment:
`split` is not really describing the parameter well. What we are setting here is the length of the spacing when the chart is split. I think `spacing` would be a better name. It would also be closer to CSS naming convention, what we try to follow in the styles.